tests/Zend/Ldap/.ci/import.txt

```diff
@@ -0,0 +1,2 @@
+ldapadd -x -D cn=admin,dc=example,dc=com -W -f basedn.ldif
```

Comment:
why in hidden dir ".ci"? A more appropriate is a subdirectory in "tests/resources".

Reply:
@glensc it depends only on OpenLDAP version (I changed to newer message on the current version).
For the moment all the code is only php 7.4 compatible. After we manage to integrate it with github actions, we will fix it for php 8.